sklearn/cluster/tests/test_optics.py

```diff
@@ -170,14 +169,14 @@ def test_reach_dists():
     # Tests against known extraction array
 
     rng = np.random.RandomState(0)
-    n_points_per_cluster = 250
-
-    C1 = [-5, -2] + .8 * rng.randn(n_points_per_cluster, 2)
-    C2 = [4, -1] + .1 * rng.randn(n_points_per_cluster, 2)
-    C3 = [1, -2] + .2 * rng.randn(n_points_per_cluster, 2)
-    C4 = [-2, 3] + .3 * rng.randn(n_points_per_cluster, 2)
-    C5 = [3, -2] + 1.6 * rng.randn(n_points_per_cluster, 2)
-    C6 = [5, 6] + 2 * rng.randn(n_points_per_cluster, 2)
+    n_points_per_cluster = 50
```

Comment:
Why do you change the test here? I'm fine with reducing `n_points_per_cluster``, but ``.8 * rng.randn(n_points_per_cluster, 2)`` -> ``8 * rng.randn(n_points_per_cluster, 2)`` seems strange. It's hard to form a cluster with your version.

Reply:
The issue was kinda related to floating point precisions, therefore we had the idea to scale up the data. But you're right, to scale the data I also need to scale the centers. I'll do so.

Reply:
How about keeping the original test? Or the original test cannot detect your problem, but the new test can?

Reply:
Also, I don't think it's good to get expected result from our implementation. Where's your expected result come from?

Reply:
The original test fails on 32 bit systems.

Reply:
The related conversation happened after this comment: https://github.com/scikit-learn/scikit-learn/pull/11857#issuecomment-414773239

Reply:
> The original test fails on 32 bit systems.

I assume this PR aims to solve the problem. Seems that I'm wrong? So it's impossible to pass the original test on 32-bit system? If so, is it possible to only reduce ``n_points_per_cluster``? If not, then I think it's acceptable to scale up the data, as long as you get expected result from another implementation (e.g., R)

Reply:
I could do that, but are you opposed to changing the `==` to `np.isclose` when comparing floats?